src/templates/zh-TW/2020/methodology.html

```diff
@@ -0,0 +1,476 @@
+
+{% block title %}統計方法 | Web Almanac 精粹至 HTTP Archive{% endblock %}
+
+{% block description %}敘述2020年Web Almanac網路年鑑如何融合：資料集和工具與專案如何運作。{% endblock %}
+
+{% block twitter_image_alt %}{{ year }}Web Almanac統計方法{% endblock %}
+
+{% block index %}
+      <ul>
+        <li><a href="#overview">總覽</a></li>
+        <li><a href="#dataset">關於資料集</a>
+          <ul>
+            <li><a href="#websites">網站</a></li>
+            <li><a href="#metrics">指標</a></li>
+          </ul>
+        </li>
+        <li><a href="#tools">工具</a>
+          <ul>
+            <li><a href="#webpagetest">WebPageTest</a></li>
+            <li><a href="#lighthouse">Lighthouse</a></li>
+            <li><a href="#wappalyzer">Wappalyzer</a></li>
+            <li><a href="#chrome-ux-report">Chrome UX Report</a></li>
+            <li><a href="#third-party-web">Third Party Web</a></li>
+            <li><a href="#rework-css">Rework CSS</a>
+              <ul>
+                <li><a href="#rework-utils">Rework Utils</a></li>
+              </ul>
+            </li>
+            <li><a href="#parsel">Parsel</a></li>
+          </ul>
+        </li>
+        <li><a href="#process">分析流程</a>
+          <ul>
+            <li><a href="#planning">規劃</a></li>
+            <li><a href="#analysis">分析</a></li>
+            <li><a href="#interpretation">說明</a></li>
+          </ul>
+        </li>
+        <li><a href="#looking-ahead">展望未來</a></li>
+      </ul>
+{% endblock %}
+
+{% block main_content %}
+    <section>
+      <!-- Show large image for large screens and high density screens and use webp when supported -->
+      <picture>
+          <source media="(min-width: 327px)" type="image/webp" srcset="/static/images/methodology-banner.webp" />
+          <source media="(min-width: 327px)" type="image/jpeg" srcset="/static/images/methodology-banner.png" />
+          <source type="image/webp" srcset="/static/images/methodology-banner.webp 2x" />
+          <source type="image/jpeg" srcset="/static/images/methodology-banner.png 2x" />
+          <source type="image/webp" srcset="/static/images/methodology-banner-sm.webp" />
+          <source type="image/jpeg" srcset="/static/images/methodology-banner-sm.png" />
+          <img src="/static/images/methodology-banner.png" class="content-banner" alt="" width="1200" height="984" loading="eager" />
+        </picture>
+        <h2 id="overview"><a href="#overview" class="anchor-link">Overview</a></h2>
+
+        <p>
+          Web Almanac網路年鑑是由<a href="https://httparchive.org">HTTP Archive</a>組織的專案。HTTP Archive在2010年由Steve Souders創辦，任務目標為追蹤網路如何建構。它按月評估數以百萬的網頁並將TB位元數據置<a href="https://httparchive.org/faq#how-do-i-use-bigquery-to-write-custom-queries-over-the-data">BigQuery</a>供大眾分析。
+        </p>
+
+        <p>
+          Web Almanac網路年鑑的任務是將網路狀態成為公開的年度知識庫。我們的目標是透過主題專家的洞察語意讓網路社群無障礙的進入HTTP Archive資料庫。
+        </p> 
+
+        <p>
+          Web Almanac網路年鑑2020年版共有四個部分：內文、體驗、發佈和傳遞。在這當中，每個章節都透過不同角度探索相關主題，例如第二部分探索使用者經驗在效能、隱私、無障礙網頁的各種角度。
```

Comment:
`publishing` 不確定翻「出版」會不會比較好，而 `distribution` 翻「分佈區域」也許比較正確，因為「傳遞」似乎是個動詞而且會有點不知道是傳遞到什麼地方 😂

另外還有最後的「的各種角度」也許翻成「等各種角度」會更通順？

```suggestion
          Web Almanac 網路年鑑 2020 年版共有四個部分：內文、體驗、出版和分佈區域。在這當中，每個章節都透過不同角度探索相關主題，例如第二部分探索使用者經驗在效能、隱私、無障礙網頁等各種角度。
```

Reply:
這句也是在指章節目錄的四大部分，publishing (CMS, Ecommerce, Jamstack)、distribution (Page Weight, Compression, Caching, Resource Hints, HTTP/2)。不太確定怎麼翻譯會比較適合?
+1 覺得'等各種角度'也挺順暢的。

Reply:
> 這句也是在指章節目錄的四大部分，publishing (CMS, Ecommerce, Jamstack)、distribution (Page Weight, Compression, Caching, Resource Hints, HTTP/2)。不太確定怎麼翻譯會比較適合?

hmm, 我如果有更好的想法會再留言 😂